src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -812,10 +812,7 @@ void Connection::handle_connection_established()
 
     m_state = ConnectionState::connected;
 
-    // TODO(RCORE-1380) get this information in-band rather than from the websocket.
-    if (auto coid = m_websocket->get_appservices_request_id(); !coid.empty()) {
```

Comment:
can we keep this for now until the server starts sending log messages? otherwise we won't be logging this at all until then